flake.nix

```diff
@@ -92,13 +92,19 @@
     in eachDefaultSystem (system:
       let pkgs = import nixpkgs { inherit system; };
       in {
-        devShell = pkgs.mkShell {
+        devShells.default = pkgs.mkShell {
           buildInputs =
             [ (pkgs.python3.withPackages (ps: with ps; [ PyGithub ])) ];
         };
-        package = { dependencyOverrides ? { }, ... }@args:
-          pkgs.callPackage self
-          (args // { dependencyOverrides = (inputs // dependencyOverrides); });
+        package = { ... }@args:
+          pkgs.lib.warn "Deprecated, please use `packages.${system}.default` instead!"
+          (pkgs.callPackage self args);
```

Comment:
No need for this merge, since we already do this here: https://github.com/nix-community/nix-doom-emacs/blob/master/default.nix#L74-L78

Reply:
This actually had the bizarre consequence of preferring the pined inputs instead of the user overrides, that is probably undesirable.